fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -4514,7 +4514,7 @@ public long getNextId() {
                     for (Partition partition : olapTable.getAllPartitions()) {
                         long partitionId = partition.getId();
                         DataProperty dataProperty = partitionInfo.getDataProperty(partition.getId());
-                        Preconditions.checkNotNull(dataProperty);
+                        Preconditions.checkNotNull(dataProperty, partition.getName() + ", pId:" + partitionId + ", db: " + dbId + ", tbl: " + tableId);
```

Comment:
The second parameter is better to be a complete sentence.

Reply:
This is not a log, just for printing some debug info. So I think it is not necessary to be a sentence.